packages/protocol/test-sol/unit/common/interfaces/IEpochManagerEnablerMock.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.5.13 <0.9.0;
+
+interface IEpochManagerEnablerMock {
+  function initEpochManager() external;
+  function captureEpochAndValidators() external;
+  function numberValidatorsInCurrentSet() external returns (uint256);
+  function numberValidatorsInSet(uint256) external returns (uint256);
+  function validatorSignerAddressFromCurrentSet(uint256 index) external returns (address);
+  function addValidator(address validator) external;
```

Comment:
I think it's better to use IEpochManager and leave the unused functions explicitly reverting as unimplemented, because if the main interface changes the one in the mock will generate a weird error in the test

Reply:
The main reason to use this interface, is to have access to the mock functions that are not included in the actual contract interface. e.g.: `addValidator()`

I could update the `IEpochManagerEnabler` to include `numberValidatorsInCurrentSet()`,  `numberValidatorsInSet()`, `validatorSignerAddressFromCurrentSet()` and move the `addValidator()` to the `TestWithUtils08` contract, but I wanted to avoid modifying the interface during code freeze.